boilingcore/boilingcore.go

```diff
@@ -156,6 +156,7 @@ func (s *State) Run() error {
 		LQ:                strmangle.QuoteCharacter(s.Dialect.LQ),
 		RQ:                strmangle.QuoteCharacter(s.Dialect.RQ),
 		OutputDirDepth:    s.Config.OutputDirDepth(),
+		UserFuncs:         s.Config.TemplateUserFuncs,
```

Comment:
I think we should call this `CustomFuncs` or `AdditionalFuncs` or something else.

Because aside from users, sqlboiler can also be used as a base for other executables that want to generate code from a DB schema. e.g. [boilingfactory](https://github.com/stephenafamo/boilingfactory)